worlds/yoshisisland/Rom.py

```diff
@@ -1157,40 +1163,41 @@ def patch_rom(world: "YoshisIslandWorld", rom: LocalRom, player: int) -> None:
         rom.write_bytes(0x010045, ([world.leader_color]))
         rom.write_bytes(0x01003D, bytearray(world.color_order))
         rom.write_bytes(0x010043, ([world.leader_color]))
-    elif world.options.yoshi_colors.value in {2, 3}:
+    elif world.options.yoshi_colors in {YoshiColors.option_random_color, YoshiColors.option_singularity}:
         rom.write_bytes(0x010000, bytearray(world.level_colors))
 
-    if world.options.minigame_checks.value in {2, 3}:
+    if world.options.minigame_checks in {YoshiColors.option_random_color, YoshiColors.option_singularity}:

```

Comment:
I don't think this is supposed to use `YoshiColors`

Reply:
Lol